zipkin-server/src/test/java/zipkin2/server/internal/eureka/BaseITZipkinEureka.java

```diff
@@ -60,7 +59,8 @@
     "zipkin.storage.type=", // cheat and test empty storage type
     "zipkin.collector.http.enabled=false",
     "zipkin.query.enabled=false",
-    "zipkin.ui.enabled=false"
+    "zipkin.ui.enabled=false",
+    "zipkin.discovery.eureka.hostname=localhost"
```

Comment:
I ran `./gradlew install` from https://github.com/line/armeria/pull/5451 and set the armeria version to what it produced (1.27.2-SNAPSHOT). Then, I removed this property and the results of the tests were expected, with my machine's hostname in the instanceId (along with the port), and my machine's hostname without the port as the vipAddress